### DISCUSSION TITLE
ccbr project join in OPA ecosystem

### DISCUSSION THREAD

**Discussion by niuzhi** [open-policy-agent/community#262] [OPA and Rego] [answered]
Hello,I developed a tool CCBR, Now I want to join OPA Ecosystem. How can I operate?

CCBR introduction. 

https://github.com/niuzhi/ccbr

CCBR is a policy management system project. It uses the policy language Rego to implement the CIS benchmark test of cloud native kubernetes. In addition, it integrates gatekeeper, manages its constraint templates, constraints and policies, and supports policy deployment and audit inspection.

![image](https://user-images.githubusercontent.com/33443724/187858408-a2f5ecd6-8ad2-43f1-9201-ae041cbfd803.png)

**Comment by srenatus** [comment, marked as answer]
Welcome! All those ecosystem tiles are generated from this yaml file: https://github.com/open-policy-agent/opa/blob/main/docs/website/data/integrations.yaml -- You are very welcome to open a PR to add your project.

**Reply by srenatus** [reply]
Can we mark this as answer? It's what should be done about this, and we're happy to merge the PR when the project is ready 🎉

**Reply by niuzhi** [reply]
@srenatus 
https://github.com/open-policy-agent/opa/pull/5084
